<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GlobalPlotSettings</class>
 <widget class="QWidget" name="GlobalPlotSettings">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="frequencySpinBox">
       <property name="decimals">
        <number>0</number>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="4">
      <widget class="QComboBox" name="plotStyleComboBox">
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentIndex">
        <number>2</number>
       </property>
       <item>
        <property name="text">
         <string>2D Polar Constant Azimuth</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2D Polar Constant Elevation</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3D Plot</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="3">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="plotStyleLabel">
       <property name="text">
        <string>Plot Style</string>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QPushButton" name="frequencyResetButton">
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Resets frequency to the initial default value</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="frequencyLabel">
       <property name="text">
        <string>Frequency</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QComboBox" name="frequencyUnits">
       <item>
        <property name="text">
         <string>Hz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>kHz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MHz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>GHz</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="manualScalingCheckBox">
       <property name="text">
        <string>Manual Scale</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QDoubleSpinBox" name="dbMaxSpinBox">
       <property name="prefix">
        <string>max=</string>
       </property>
       <property name="suffix">
        <string>dB</string>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>-300.000000000000000</double>
       </property>
       <property name="maximum">
        <double>300.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>5.000000000000000</double>
       </property>
       <property name="value">
        <double>20.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QDoubleSpinBox" name="dbIncSpinBox">
       <property name="prefix">
        <string>inc=</string>
       </property>
       <property name="suffix">
        <string>dB</string>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>5.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="scaleResetButton">
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="dbMinSpinBox">
       <property name="prefix">
        <string>min=</string>
       </property>
       <property name="suffix">
        <string>dB</string>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>-300.000000000000000</double>
       </property>
       <property name="maximum">
        <double>300.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>5.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="4">
      <widget class="QLabel" name="errorMsg">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>10</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>15</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string>ERROR</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
